### ISSUE TITLE
Luxembourg is missing from the list

### ISSUE THREAD

**Issue by astuanax** [w3c/wai-policies-prototype#423]
More details can be found here:

https://luxembourg.public.lu/en/support/accessibilite.html
or the PR:
https://gouvernement.lu/fr/actualites/toutes_actualites/communiques/2019/06-juin/24-hansen-digitale.html

Date: 28tyh of Mai 2019
Link to legislation: https://legilux.public.lu/eli/etat/leg/loi/2019/05/28/a373/jo
Type: Accessibility law
Scope: public sector
Web only: no
Level: WCAG 2.1 AA

**Comment by astuanax** [comment]
Seems there is already a pull request on the list: 

https://github.com/w3c/wai-policies-prototype/pull/413
